docs/Users_Guide/release-notes.rst

```diff
@@ -9,226 +9,428 @@ When applicable, release notes are followed by the GitHub issue number which des
 enhancement, or new feature (`MET GitHub issues <https://github.com/dtcenter/MET/issues>`_).
 Important issues are listed **in bold** for emphasis.
 
-MET Version 12.0.0-rc1 Release Notes (20241114)
------------------------------------------------
+MET Version 12.1.0-beta1 Release Notes (20250124)
+-------------------------------------------------
 
   .. dropdown:: Bugfixes
 
+     * Bugfix: Fix Grid-Stat segfault when SEEPS is the only NetCDF output type requested
+       (`#3020 <https://github.com/dtcenter/MET/issues/3020>`_).
+
+  .. dropdown:: Enhancements
+
+     * Enhance ASCII2NC to read USCRN point observations
+       (`#1019 <https://github.com/dtcenter/MET/issues/1019>`_).
+     * Enhance Multivariate MODE to support multiple convolution radii and thresholds
+       (`#2709 <https://github.com/dtcenter/MET/issues/2709>`_).
+     * Enhance Gen-Vx-Mask to support a new local solar time masking region option
+       (`#2966 <https://github.com/dtcenter/MET/issues/2966>`_).
+     * **Create a new Pair-Stat tool to compute statistics for already paired forecast and observation data**
+       (`#3006 <https://github.com/dtcenter/MET/issues/3006>`_).
+     * Develop a class that supports reading both FCST and OBS from a IODA file to use with the new Pair-Stat tool\
+       (`#3007 <https://github.com/dtcenter/MET/issues/3007>`_).
+     * Enhance the Grid-Stat GRAD line type with additional gradient vector-based statistics to measure sharpness
+       (`#3024 <https://github.com/dtcenter/MET/issues/3024>`_).
+     * Enhance Series-Analysis to compute statistics from the GRAD line type
+       (`#3030 <https://github.com/dtcenter/MET/issues/3030>`_).
+     * Improve MET library logging for NetCDF level dimension matches
+       (`#3038 <https://github.com/dtcenter/MET/issues/3038>`_).
+     * Enhance the unstructured grid library to correctly parse the LFRic model initialization time
+       (`#3056 <https://github.com/dtcenter/MET/issues/3056>`_).
+
+  .. dropdown:: Documentation
+
+     * Update the MET User's Guide to specify the data types in each output line type table
+       (`#3032 <https://github.com/dtcenter/MET/issues/3032>`_).
+
+  .. dropdown:: Repository, build, and test
+
+     * Create a new test utility to search a 3D cloud of points for the nearest neighbor to a specified (lat, lon, elev) location
+       (`#3012 <https://github.com/dtcenter/MET/issues/3012>`_).
+     * Create a new development environment file for Seneca for the Intel oneAPI compilers
+       (`#3047 <https://github.com/dtcenter/MET/issues/3047>`_).
+     * Decouple the MET unit tests from the MET installation location
+       (`#3051 <https://github.com/dtcenter/MET/issues/3051>`_).
+
+MET Version 12.0.0 Release Notes (20241218)
```

Comment:
I suggest removing the "MET Version 12.0.0 Release Notes (20241218)" title and everything below to be consistent with the other components' release notes.  The [Release Notes Instructions for developmental releases](https://metplus.readthedocs.io/en/develop/Release_Guide/met_development.html#update-release-notes) say that if you are creating a beta1 release, remove the previous version’s release notes.